resources/events.hubdb.json

```diff
@@ -23,24 +24,39 @@
       "width": 206,
       "type": "IMAGE"
     },
+    {
+      "name": "limitless_event_capacity",
```

Comment:
I wonder if it would be better to reverse this and have "limited_capacity" as the new column which then is tied to `event_capacity` and `registered_attendee_count`.

Reply:
One note here: I've changed the name of the column to unlimited_event_capacity and pushed this change. 

My thought process here: The default for events is to have specified event capacity. If a user wants to have unlimited capacity (ie, change the default), they need to check the unlimited event capacity checkbox. To me, it made more sense that a user needs to take action to change the default setting.

Reply:
When it comes to the HubDB UI, I think we're feeling the pain on of not having a row-based interface with fields that are required. That said, I don't think defaulting to no limits on capacity would be bad. My suggestion was more around how code using the column reads and the mental model. It feels more natural to say "I want to limit attendees to this event" instead of "I want this event to have no limit on the number of attendees." The logic is also more natural to me:


```js
if (event.limited_capacity && event.registered_attendee_count < event.event_capacity) {
  // 
}
```

vs. 

```js
if (!event.unlimited_event_capacity && event.registered_attendee_count < event.event_capacity {

}
```

Reply:
@gcorne, can we discuss this in more detail in our meeting today? The logic surrounding limited vs unlimited event capacity is unexpectedly confusing to me.